src/util/system.cpp

```diff
@@ -1163,8 +1163,9 @@ void ScheduleBatchPriority()
 {
 #ifdef SCHED_BATCH
     const static sched_param param{};
-    if (pthread_setschedparam(pthread_self(), SCHED_BATCH, &param) != 0) {
-        LogPrintf("Failed to pthread_setschedparam: %s\n", strerror(errno));
+    int rc;
+    if ((rc = pthread_setschedparam(pthread_self(), SCHED_BATCH, &param)) != 0) {
```

Comment:
nit: could look clearer
```suggestion
    const int rc = pthread_setschedparam(pthread_self(), SCHED_BATCH, &param);
    if (rc != 0) {
```

...  until switching to C++17 :)

Reply:
Hi, @hebasto

Thank you for suggesting change.
I accept your suggested change.